build/release/script/announce.sh

```diff
@@ -15,17 +17,31 @@
 # limitations under the License.
 #
 
-Title: [ANNOUNCE] Apache Kyuubi (Incubating) released {release_version}
+set -o pipefail
+set -e
 
-Content:
-Hi all,
+RELEASE_DIR="$(cd "$(dirname "$0")"/..; pwd)"
+
+######### Please modify the variables ##########
+release_version=v1.4.0-incubating
```

Comment:
How to change it to use env var?

Reply:
I will leave them be empty by defaults.

Reply:
it is not convenient to use env var, because they are specified.

Reply:
I prefer that the release manager modify the variable value in this script

Reply:
have refactored it